### ISSUE TITLE
Search Boards screen missing contrast

### ISSUE THREAD

**Issue by MaximKipot** [grafana/grafana#29714]
Dear Grafana Community / Team,

unfortunately there are still some minor accessibility issues. Please refer to the screenshot to see all covered usecases.

**Steps to reproduce**:

- click on Home on the top left and get the results with all the boards and folders

**Actual Result**:
Dark Theme:
<img width="1351" alt="Screenshot 2020-12-09 at 12 58 39" src="https://user-images.githubusercontent.com/1902515/101628264-d5c84080-3a1f-11eb-8420-0c5ddbb176ee.png">
Light Theme:
<img width="1350" alt="Screenshot 2020-12-09 at 13 26 59" src="https://user-images.githubusercontent.com/1902515/101630223-df9f7300-3a22-11eb-9391-12ba3ed08530.png">

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion]
https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html 
&
https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-contrast

**Environment**:
- Grafana version: 7.4.0-pre
- Data source type & version: None
- User OS & Browser: MacOS Catalina, Mozilla and Chrome

**Comment by ckbedwell** [comment]
Update on this as of Grafana 10.2.0:

## Light theme

**Failures**
- input placeholder [contrast: 3.04](https://www.siegemedia.com/contrast-ratio#rgba%2836%2C%2041%2C%2046%2C%200.5%29-on-%23fff) (should be at least 4.5)
![Screenshot of Grafana input component for searching dashboards on the Explore page.](https://github.com/grafana/grafana/assets/6906380/38ca380c-9fb3-4ccb-b8ed-a3bc3c45f71b)

**Passes**
- The parent folder text on the white background has a [contrast of 6.48](https://www.siegemedia.com/contrast-ratio#rgba%2836%2C%2041%2C%2046%2C%200.75%29-on-%23fff) 
- The parent folder text within the card has a [contrast of 6.17](https://www.siegemedia.com/contrast-ratio#rgba%2836%2C%2041%2C%2046%2C%200.75%29-on-rgb%28244%2C%20245%2C%20245%29).


## Dark theme

**Passes**
- The input placeholder text has a contrast of [4.89](https://www.siegemedia.com/contrast-ratio#%23ccccdc99-on-%23111217)
- The parent folder text on the dark grey background has a [contrast of 5.32](https://www.siegemedia.com/contrast-ratio#rgba%28204%2C%20204%2C%20220%2C%200.65%29-on-rgb%2824%2027%2031%29)
- The parent folder text within the card has a [contrast of 4.69](https://www.siegemedia.com/contrast-ratio#rgba%28204%2C%20204%2C%20220%2C%200.65%29-on-%23282b31).
